tests/utils.test.ts

```diff
@@ -81,17 +126,173 @@ t.test("utils", t => {
       nested: {
         nested2: {
           nested3: {
-            bar: "baz"
-          }
+            bar: "baz",
+          },
         },
         null: null,
-        noop: () => null
-      }
+        noop: () => null,
+      },
     };
 
     const flattened = flattenObject(myObject);
 
     t.equal((flattened as any).foo, "bar");
     t.equal(flattened["nested.nested2.nested3.bar"], "baz");
   });
+
+  t.test("should get nanoseconds time", t => {
+    t.plan(2);
+
+    t.test("hrtime not defined", async t => {
+      t.plan(1);
+
+      if (typeof globalThis.process === "undefined") {
+        t.skip("This test is only for Node.js");
+        return;
+      }
+
+      const performance = globalThis.performance;
+      const hrtime = globalThis.process.hrtime;
+
+      // @ts-expect-error - we are testing this
+      globalThis.process.hrtime = undefined;
```

Comment:
I don't like it at all. I tried mocking the `utils.js` module, but I think there is a problem between resolvers and mocks. The module is never found. I would like to open an issue for this.